macOS/DuckDuckGo/BookmarksBar/View/BookmarksBarViewController.swift

```diff
@@ -49,8 +49,8 @@ final class BookmarksBarViewController: NSViewController {
     private var dragDestination: (folder: BookmarkFolder, mouseLocation: NSPoint, hoverStarted: Date)?
 
     fileprivate var clipThreshold: CGFloat {
-        let viewWidthWithoutClipIndicator = view.frame.width - clippedItemsIndicator.frame.minX
-        return view.frame.width - viewWidthWithoutClipIndicator - 3
+        let indicatorFrameInCollectionView = bookmarksBarCollectionView.convert(clippedItemsIndicator.frame, from: clippedItemsIndicator.superview)
+        return indicatorFrameInCollectionView.minX - 3
```

Comment:
The calculation was based on the width of `view` instead of basing it on the width of `bookmarksBarCollectionView`.  This was ignoring the left-side padding.

Additionally, the `clippedItemsIndicator`'s `minX` value needs to be calculated in relation to `bookmarksBarCollectionView`'s frame.